README.md

```diff
@@ -154,19 +154,7 @@ Currently most of the documentation on customization is given by the examples.
 
 ### other templates
 
-* `nbImage`: image command to show images (see `penguins.nim` example linked above)
-* `nbFile`: content (string or untyped) is saved to file (see example document [files](https://pietroppeter.github.io/nimib/files.html))
-* `nbShow`: show a variable that has a `toHtml` proc defined. For example to pretty print a dataframe.
-* `nbRawHtml`: called with string content, it will add the raw content to document (html backend)
-* `nbTextWithCode`: a variant of `nbText` that also reads nim source. See example of usage
-  at the end of the source in `numerical.nim` linked above.
-* `nbCodeSkip`: a variant of `nbCode` that that displays highlighted code but does not compile or run it.
-* `nbCapture`: a block that only shows the captured output of a code block
-* `nbPython`:  can be used after calling `nbInitPython()` and it runs and capture output of python code;
-  requires [nimpy](https://github.com/yglukhov/nimpy).
-* `nbClearOutput`: clears the output of preceding code block,
-  useful in case a previous command has produced output that you do not want to show for some reason.
-
+You can find a complete description of the code blocks along with examples in [allblocks.nim](https://pietroppeter.github.io/nimib/allblocks.html)
```

Comment:
as mentioned in the contributing.md, you need to change index.nim to modify the readme

Reply:
I changed the index.nim file too.

Reply:
I made one extra change in the index.nim file though. I changed the title to
```
### other blocks
````